Sources/ComposableArchitecture/Internal/Deprecations.swift

```diff
@@ -3,6 +3,20 @@ import Combine
 import SwiftUI
 import XCTestDynamicOverlay
 
+// NB: Deprecated after 0.34.0:
+
+extension Effect {
+  @available(
+    *,
+    deprecated,
+    message: "For improved safety, using a variadic list is no longer supported. Use an array of identifiers instead. For more on this change, see: https://github.com/pointfreeco/swift-composable-architecture/pull/1041"
```

Comment:
@stephencelis are you fine with this? The plan is to add a some context to the PR outlining the issue related to supporting both `AnyHashable...` and `[AnyHashable]`. Not sure if safety is the right word though.

Reply:
Good change. I'll update the PR description tonight.